recipe/meta.yaml

```diff
@@ -64,8 +64,9 @@ test:
     - rio info {{ RECIPE_DIR }}/test_data/test.tif  # [not win]
     - rio info {{ RECIPE_DIR }}\\test_data\\test.tif  # [win]
     # test_target_aligned_pixels is failing for all OSes for proj 8.0.0
-    - $PYTHON -m pytest -v -m "not wheel" -rxXs -k "not test_target_aligned_pixels" tests  # [linux and target_platform != "linux-aarch64"]
-    - $PYTHON -m pytest -v -m "not wheel" -rxXs -k "not (test_target_aligned_pixels or test_outer_boundless_pixel_fidelity)" tests  # [linux and target_platform == "linux-aarch64"]
+    # skip linux tests for cross-compiled targets
+    - $PYTHON -m pytest -v -m "not wheel" -rxXs -k "not test_target_aligned_pixels" tests  # [linux and build_platform == target_platform]
+    - $PYTHON -m pytest -v -m "not wheel" -rxXs -k "not (test_target_aligned_pixels or test_outer_boundless_pixel_fidelity)" tests  # [linux and build_platform == target_platform]
```

Comment:
@conda-forge/rasterio, assuming this works, let me know if you're okay with it.

Reply:
I don't fully understand why this is necessary.  It seems like tests should be skipped when cross-compiling because of:
https://github.com/conda-forge/rasterio-feedstock/blob/995b626e70802dd4772471e90f6a178817c54589/conda-forge.yml#L11